### PR TITLE
Speed up train_model task: extra model age check

### PR DESCRIPTION
**Pull request by MartijnCa**

### CODE REVIEW

File: openstef/tasks/train_model.py [hunk 1]
```diff
@@ -84,13 +90,30 @@ def train_model_task(
 
     context.perf_meter.checkpoint("Added metadata to PredictionJob")
 
+    # Check the model age before retrieving the input data to speed up train job.
+    # (The exact same model age check is also part of the "train_model_pipeline".)
```
Comment (FrankKr) [inline openstef/tasks/train_model.py:94]: Useful to add this comment to prevent confusion:)

**Review by FrankKr — APPROVED** [review]
Looks good. Single minor suggestion

File: openstef/tasks/train_model.py [hunk 2]
```diff
@@ -84,13 +90,30 @@ def train_model_task(
...
+    serializer = MLflowSerializer(mlflow_tracking_uri=mlflow_tracking_uri)
+
+    # Get old model and age
+    _, _, old_model_age = train_pipeline_step_load_model(pj, serializer)
+
+    # Check old model age and continue yes/no
+    if (old_model_age < MAXIMUM_MODEL_AGE) and check_old_model_age:
+        context.logger.warning(
+            f"Old model is younger than {MAXIMUM_MODEL_AGE} days, skip training"
+        )
```
[the remark this answers is not in the document]
Comment (FrankKr) [reply openstef/tasks/train_model.py:106]: using checkpoints makes is a bit easier to track the flow of the code in the logging.

[the remark this answers is not in the document]
Comment (MartijnCa) [reply openstef/tasks/train_model.py:106]: Thanks for the suggestion, cool feature I did not know about 👍

**Pull request merged**